examples/anonymize_dataset.py

```diff
@@ -76,7 +76,7 @@
 # Write experimental MEG data, fine-calibration and crosstalk files
 write_raw_bids(
     raw=raw, bids_path=bids_path, events_data=events_path, event_id=event_id,
-    empty_room=bids_path_er, verbose=False
+    er_bids_path=bids_path_er, verbose=False
```

Comment:
er_bids_path=bids_path_er

hum... one of the 2 needs to be changed so it's consistent